1-js/02-first-steps/07-type-conversions/article.md

```diff
@@ -34,7 +34,7 @@ String conversion is mostly obvious. A `false` becomes `"false"`, `null` becomes
 
 ## Numeric Conversion
 
-Numeric conversion happens in mathematical functions and expressions automatically.
+Numeric conversion in mathematical functions and expressions happen automatically.
```

Comment:
Numeric 
```suggestion
Numeric conversion happens automatically in mathematical functions and expressions.
```
warning I am not a native eng speaker

Reply:
@joaquinelio your version looks good too